README.md

```diff
@@ -54,6 +54,54 @@ something similar to (the exact tasks numbers may differ):
   NOTE: Tasks Summary: Attempted 2532 tasks of which 2524 didn't need to be rerun and all succeeded.
   ```
 
+### Building for different targets
+
+Note, that Zarhus OS is supported by several targets. You can find supported
+targets in every `conf/machine` directories for every BSP layer.
+
+To change the target you are building your image for, you can edit the `machine`
```

Comment:
Editing a file and building from `-dirty` tree is a suboptimal approach.

Can we instead specify target as a command line option? Either override the default value from .yml file, or remove the target from the .yml file and instruct in the docs that it should be always provided via cmd line?

Reply:
I will check that.

Reply:
We can override at least `KAS_DISTRO, `KAS_MACHINE`, `KAS_TARGET`, `KAS_TASK`.
Overriding machine variable was mentioned in this PR here: https://github.com/zarhus/meta-zarhus/blob/c34c57099a0a9abb026f8376cbcff99dfc79ebbf/README.md?plain=1#L98

Reply:
Addressed [here](https://github.com/zarhus/meta-zarhus/compare/c34c57099a0a9abb026f8376cbcff99dfc79ebbf..f247a584d112ff6335f21347f85fc838b7bb34a8)

Reply:
Why did you remove `zarhus-machine-cm3.conf`?

Reply:
Because it is surplus and unused.

Reply:
But I have forgotten to remove it from recipes :man_facepalming:

Reply:
Then remove all mentions of this target?

```
$ grep -R zarhus-machine-cm3
meta-zarhus-bsp/meta-zarhus-bsp-rockchip/recipes-kernel/linux/linux-yocto_%.bbappend:COMPATIBLE_MACHINE:zarhus-machine-cm3 = "zarhus-machine-cm3"
README.md:  $ cd build/tmp/deploy/images/zarhus-machine-cm3
README.md:  $ sudo bmaptool copy zarhus-base-image-zarhus-machine-cm3.wic.gz /dev/sdx
README.md:  zarhus-base-image-zarhus-machine-cm3.wic.bmap zarhus-base-image-zarhus-machine-cm3.wic.gz /dev/sdx
README.md:  bmaptool: info: copying image 'zarhus-base-image-zarhus-machine-cm3.wic.gz' to block device '/dev/sdx' using bmap file 'zarhus-base-image-zarhus-machine-cm3.wic.bmap'
```

Reply:
Yep. fixing it...

Reply:
Added to `zarhus/docs` here https://github.com/zarhus/docs/pull/11.